package.json

```diff
@@ -1865,6 +1865,12 @@
                     "default": false,
                     "description": "Determines if pylance's experimental notebook support is used or not",
                     "scope": "machine"
+                },
+                "jupyter.completionTriggerCharacters": {
+                    "type": "string",
+                    "default": ".%",
```

Comment:
Isn't this specific to just python.

Reply:
Everything is specific to python. We only register a completion provider for python cells.

Reply:
In that case shouldn't this setting be named `jupyter.pythonCOmpletionTriggerCharacters`
As this doesn't apply to Julia or R

Reply:
ignore my typos